pkg/networkmanager/firewall-client.js

```diff
@@ -327,4 +341,70 @@ firewall.removeServiceFromZones = (zones, service) => {
     return Promise.all(zones.map(z => firewall.removeService(z, service)));
 };
 
+firewall.activateZone = (zone, interfaces, sources) => {
```

Comment:
OOI, what is a "source"? An IP mask like "10.0.0.0/8"?

Reply:
yup, exactly